less/components/expander.less

```diff
@@ -36,7 +36,12 @@
     max-height: 0;
     transition: max-height .3s ease;
     .items-margin {
-      margin: 1.8rem;
+      margin: 2.5rem 4rem 3rem 4rem;
+    }
+    p, ul {
```

Comment:
I'm not a huge fan of this.

It would be unexpected to me if expander starts changing the children, and if I try to use it later and I don't want the css, I'm more likely to over ride it with more css than understand these and adapt them.